huxley/www/js/stores/CurrentUserStore.js

```diff
@@ -36,13 +38,32 @@ class CurrentUserStore extends Store {
 
   updateSchool(schoolID, delta) {
     ServerAPI.updateSchool(schoolID, delta);
-    const user = this._currentUser.school;
+    const user = this._currentUser;
     this._currentUser = {
       ...user,
       school: {...user.school, ...delta},
     };
   }
 
+  updateUser(userID, delta) {
+    $.ajax({
```

Comment:
Add this as a function on ServerAPI, instead of doing the raw async request here.